packages/nx/src/config/configuration.ts

```diff
@@ -26,3 +23,5 @@ export function workspaceLayout(): { appsDir: string; libsDir: string } {
     libsDir: nxJson.workspaceLayout?.libsDir ?? 'libs',
   };
 }
+
+export { readNxJson } from './nx-json';
```

Comment:
actually, we need it otherwise the Lerna e2e tests would fail. it is used in https://github.com/lerna/lerna/blob/main/libs/commands/run/src/index.ts#L308